### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by morpheus-87** [shevabam/ezservermonitor-web#79]
Would it be possible to make a new release? The last one is from 2015 :see_no_evil:

**Comment by pucelev** [comment]
I agree.  With all these changes, a new release would be great.

**Comment by morpheus-87** [comment]
What about this topic, @shevabam ?

**Comment by Villantiqua** [comment]
Yes, please, an update for this fine piece of software. E.g., for displaying the newest (ARM) CPUs correctly.

**Comment by artecnico** [comment]
Pleeeeease! @shevabam
